jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/FormFieldsSection.java

```diff
@@ -0,0 +1,31 @@
+package io.github.com.pages.sections;
+
+import com.epam.jdi.light.angular.elements.complex.*;
```

Comment:
Importing asterisks does not provide much visibility into which types you are using in your code.
This is inconvenient to understand the codebase.

Reply:
Updated.